### PR TITLE
Allow only fully qualified schema ID

### PR DESCRIPTION
**Pull request by jakubkoci**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ After some initial discussion between ABSA and DIDx regaring development of a Tr
 
 For more detailed information regarding the current discussion and implementation requirements read the [HackMD Write Up](https://hackmd.io/heOazbtvSi2B18I7YWkGIg?view)
 
+## Schemas
+
+We require fully qualified schemas. You can find how the unqualified should be transfromed to fully qualified and vice-versa at the following links:
```
Comment (niall-shaw) [inline README.md:13]:
```suggestion
We require fully qualified schemas. You can find how the unqualified should be transformed to fully qualified and vice-versa at the following links:
```

typo

File: src/schema/service.ts [hunk 2]
```diff
@@ -48,30 +52,26 @@ async function getAllSchemas(repository: SchemaRepository) {
 
 async function loadSchemas(
```
Comment (niall-shaw) [inline src/schema/service.ts:53]: Should we not also attempt to resolve the fully qualified schema?

Comment (jakubkoci) [reply src/schema/service.ts:53]: Eventually yes, but now it would require a bit more work and I don't think it's so benefical.

Comment (jakubkoci) [reply src/schema/service.ts:53]: Also slowing the re-deployment... so maybe when we have running service

**Review by niall-shaw — APPROVED** [review]

**Pull request merged**